lib/puppet/provider/cpdb_create/podman.rb

```diff
@@ -0,0 +1,42 @@
+Puppet::Type.type(:cpdb_create).provide(:podman) do
+
+  commands :podman => '/bin/podman'
+
+  def create
+    create_database
+    write_done_file
+  end
+
+  def exists?
+    File.exist?(done_file)
+  end
+
+  private
+
+  def create_database
+    podman(
+      "run",
+      "--network=host",
+      "quay.io/ehelms/candlepin:4.3.12",
```

Comment:
Note this version will need to be parameterized here and in the `.container` file